assets/javascripts/comments.js

```diff
@@ -75,31 +71,30 @@ function updateComment(form) {
   displayElements([textElement, form.applyChanges, form.discardChanges], 'none');
   markdownElement.style.display = '';
   markdownElement.innerHTML = '<em>Loading…</em>';
-  $.ajax({
-    url: url,
-    method: 'PUT',
-    data: $(form).serialize(),
-    success: () => {
-      $.ajax({
-        url: url,
-        method: 'GET',
-        success: response => {
+  fetchWithCSRF(url, {method: 'PUT', body: new FormData(form)})
+    .then(response => {
+      if (!response.ok) throw `Server returned ${response.status}: ${response.statusText}`;
```

Comment:
To be honest I would rather use a more specific error. like `Request failed to update comment ${response.status}: ${response.statusText}`.

Reply:
I agree. For debugging context will be useful to preserve here.

Reply:
But we do have this here. You have to look at the `.catch()`:
```js
    .catch(error => {
      window.alert(`The comment couldn't be updated : ${error}`);
    });

Reply:
Right, then we really don't need it. I suppose we have to get used to this `throw ->carch(…)` pattern.